### PR TITLE
Web app sample - changes for tutorial

### PR DESCRIPTION
**Pull request by diberry**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x ] Other... Please describe: visually highlight rewardActionId and remove _au from features passed to rank call
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by diberry** [comment]
@mandyshieh @tyclintw please review
@edjez This sample doesn't call reward. Is there a functional or conceptual reason for this? I'm not sure how instructive the sample is. Thoughts?

File: samples/HttpRequestFeatures/HttpRequestFeaturesExample/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -42,7 +42,7 @@
         @RenderBody()
         <hr />
         <footer>
-            <p>&copy; 2019 - Http Request Features Example</p>
+            <p>&copy; 2020 - Http Request Features Example</p>
```
Comment (tyclintw) [inline samples/HttpRequestFeatures/HttpRequestFeaturesExample/Views/Shared/_Layout.cshtml]: >2020 [](start = 22, length = 4)

To ensure this stays current please use @DateTime.Now.Year  #Closed

**Review by tyclintw — APPROVED** [review]
:shipit:

**Comment by mandyshieh** [comment]
LGTM

**Review by mandyshieh — APPROVED** [review]

**Comment by diberry** [comment]
@tyclintw if you are happy with it, can you please merge #sign-off - not sure what your usual process is for this.

**Pull request merged**